automate.sh

```diff
@@ -1,45 +1,45 @@
 #!/bin/bash
 
-# create the Templates directory in the home folder, if it does not exist
 if [ -f Templates.tar.gz ];
 then
 	# extract the dummy files
-	echo "Extracting.."
+	echo "Extracting files..."
 	tar -xvzf Templates.tar.gz
 
-	# check if ~/Templates already exists
-	if [ ! -d ~/Templates ];
-	then
-		mv Templates ~/Templates
-		echo -e "\nTemplates directory created.."
-	else
-		# directory exists; merge, instad
-		rsync -a Templates/ ~/Templates/
-		rm -rf Templates
+	# if .config/user-dirs.dirs does not exist, create it and append config settings to tell nautilus about our templates and
+	# create the custom Templates directory in the home folder, if it does not exist
+	if [[ ! -f ~/.config/user-dirs.dirs ]];
+		then
+			xdg-user-dirs-update
+			echo "Config file created..."
+		else
+			# this config already exists; append the entry
+			source ~/.config/user-dirs.dirs
+			customPath=$XDG_TEMPLATES_DIR
```

Comment:
Where is `$XDG_TEMPLATES_DIR` being set? 
See comment to the left, too. :)

Reply:
I am not sure if I understand your question.

If you run `xdg-user-dirs-update` the `~/.config/user-dirs.dirs` file is being created. It contains a few paths for your system, which can be called using `$XDG_TEMPLATES_DIR` after using source to let bash know that it must look at the specified file
`source ~/.config/user-dirs.dirs`.

This value, which is created by `xdg-user-dirs-update` will vary depending on the language your system is set to. On my german system it says 

> XDG_TEMPLATES_DIR="$HOME/Vorlagen"

In my english system it says

> XDG_TEMPLATES_DIR="$HOME/Templates"

That translation is automatically being done by `xdg-user-dirs-update`. So basically what I do is to check whether the